WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -102,7 +103,11 @@ class MediaLibraryViewController: UIViewController {
 
         registerChangeObserver()
 
-        updateViewState(for: pickerDataSource.totalAssetCount)
+        if isLoading {
+            updateNoResultsForFetching()
+        } else {
+            updateViewState(for: pickerDataSource.totalAssetCount)
```

Comment:
Hmm, should we combine these two methods some how? Perhaps `updateNoResultsForFetching()` is only ever called, if needed, within the call to `updateViewState(for:)`.